src/main/java/sirius/web/data/XLSProcessor.java

```diff
@@ -48,7 +48,7 @@
 
     @Override
     public void run(RowProcessor rowProcessor, Predicate<Exception> errorHandler) throws Exception {
-        Workbook wb = xslx ? new XSSFWorkbook(input) : new HSSFWorkbook(input);
+        Workbook wb = xslx ? StreamingReader.builder().bufferSize(4096).open(input) : new HSSFWorkbook(input);
```

Comment:
Reichen die per Default gesetzten 10 Rows im Cache? Siehe rowCacheSize(), https://github.com/monitorjbl/excel-streaming-reader#usage

Reply:
Dass sollte für unser Szenario in S2 passen. Das einlesen ist eh um Hausnummern schneller als die anschließende Verarbeitung. Buffer Size hab ich ein bisschen höher als default um I/O zu entlasten. Optimiert ist an den Parametern nix, aber im Test wir bekommen keinen OutOfMemoryError mehr. Das war das Ziel ;-)